### ISSUE TITLE
[Feature Request] Artist image/icons

### ISSUE THREAD

**Issue by SpitFire-666** [chr56/Phonograph_Plus#158]
Hi, currently the Artist images are album art (if available). 

![image](https://github.com/chr56/Phonograph_Plus/assets/38451588/97a1a4d8-c024-4185-9bab-22b985701fb9)

This can be confusing since some artists are on the same compilation. Also, it can only show one album. I would rather not have an image shown OR have them all set to a more appropriate stock artist icon like iTunes does 🎙️

![image](https://github.com/chr56/Phonograph_Plus/assets/38451588/f2cd8502-69a9-4d7a-9b5c-a007d30b856d)

Cheers

**Comment by rahulyadav07** [comment]
@SpitFire-666 can you please assign this enhancement to me ?

**Comment by chr56** [comment]
> @SpitFire-666 can you please assign this enhancement to me ?

Assign to you?

**Comment by rahulyadav07** [comment]
i mean to say can i implement that  or someone is working on that ?

**Comment by chr56** [comment]
> i mean to say can i implement that or someone is working on that ?

I am not working on it. _I may be busy for next 3 months_.

If you would like to contribute and be willing to implement this, this issue assigns to you.

**Comment by rahulyadav07** [comment]
Hello @chr56 
Today i started worked on this Enhancement while doing code . 

I saw `./idea ` file is not in gitIgnore because of that there some unwanted changes are coming in git .

![image](https://github.com/chr56/Phonograph_Plus/assets/63515254/9b62ce64-e239-48b3-b55d-dd92d4bb592e)

` **./idea**`  code is auto genereated by system based on own specification . 

So can i put in gitignore  ? or there is some reason to not put in gitIgnore ?

**Comment by chr56** [comment]
> So can i put in gitignore ? or there is some reason to not putting in gitIgnore ?

Yes, put `.idea/appInsightsSettings.xml` into `.idea/.gitignore`. We only tracks `codeStyles`, `runConfigurations`, `inspectionProfiles` and so on in git.
